rebasehelper/logger.py

```diff
@@ -30,47 +32,79 @@ class LoggerHelper(object):
 
     @staticmethod
     def get_basic_logger(logger_name, level=logging.DEBUG):
-        """
-        Sets-up a basic logger without any handler
+        """Sets up a basic logger without any handler.
+
+        Args:
+            logger_name (str): Logger name.
+            level (int): Severity level set to the logger.
+        Returns:
+            logging.Logger: Created logger.
 
-        :param logger_name: Logger name
-        :param level: severity level
-        :return: created logger
         """
         basic_logger = logging.getLogger(logger_name)
         basic_logger.setLevel(level)
         return basic_logger
 
     @staticmethod
-    def add_stream_handler(logger_object, level=None):
-        """
-        Adds console handler with given severity.
+    def add_stream_handler(logger_object, level=None, formatter_object=None):
+        """Adds console handler with given severity.
```

Comment:
Adds it where? Also why `console handler` when the method is called `add_stream_handler`?